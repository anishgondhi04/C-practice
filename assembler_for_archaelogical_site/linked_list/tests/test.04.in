2
list 0 new
list 1 new
list 0 build 6 alpha beta gamma delta beta gamma 
list 1 build 3 beta gamma delta 
list 0 overlap 1
list 1 overlap 0
end

