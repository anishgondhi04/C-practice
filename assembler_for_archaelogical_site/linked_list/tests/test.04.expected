List 0: new list
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 0: build
 [ alpha beta gamma delta beta gamma ] (6 items)
List 1: build
 [ beta gamma delta ] (3 items)
List 0: Overlap between lists 0 and 1 is 3
List 1: Overlap between lists 1 and 0 is 0
